pkg/loki/modules.go

```diff
@@ -319,6 +319,7 @@ func (t *Loki) initDistributor() (services.Service, error) {
 		t.Overrides,
 		prometheus.DefaultRegisterer,
 		t.Cfg.MetricsNamespace,
+		t.Tee,
```

Comment:
Where is the implementation and initialization for the Tee module here?

Reply:
No implementation here yet, we need this hook for an enterprise feature.